target/scripts/start-mailserver.sh

```diff
@@ -1147,6 +1147,51 @@ function _setup_ssl
 {
   _notify 'task' 'Setting up SSL'
 
+  # Primary certificate to serve for TLS
+  # NOTE: The `sed` substituion delimiter uses `~` instead of `/` due to file paths as values
+  function _set_certificate()
```

Comment:
Everything looks good to me when it comes to Bash style. I guess there is one or more commit coming - therefore changing `function _set_certificate()` to `function _set_certificate` can be implemented in one of the following commits. I know, nip-picky here - but these scripts need great attention :)